package/MDAnalysis/analysis/hydrogenbonds/hbond_analysis.py

```diff
@@ -260,6 +272,9 @@ def __init__(self, universe,
         """Set up atom selections and geometric criteria for finding hydrogen
         bonds in a Universe.
 
+        Hydrogen bond selections may be achieved with either a resname, atom 
+        name combination, or when those are absent, atom types.
+
```

Comment:
reference the `donor_sel` and `acceptors_sel` keywords
```suggestion
        Hydrogen bond selections with `donors_sel` , `hydrogens_sel`, and
        `acceptors_sel` may be achieved with either a *resname*, atom *name* 
        combination, or when those are absent, with atom *type* selections.
        
```
I also emphasized the selection keywords.